spec/services/atmosphere/cloud/satisfy_appliance_spec.rb

```diff
@@ -17,6 +17,73 @@
   let!(:openstack) { create(:openstack_with_flavors, funds: [fund]) }
   let!(:tmpl_of_shareable_at) { create(:virtual_machine_template, appliance_type: shareable_appl_type, tenants: [openstack])}
 
+  context '#assign fund' do
+    let!(:t) { create(:openstack_with_flavors, active: true, funds: [fund]) }
+    let!(:vmt) { create(:virtual_machine_template, tenants: [t]) }
+    let!(:appliance_type) { create(:appliance_type, preference_cpu: 0, \
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.